backend/Kodstar-Issue_Tracker_3-1.0.1-swagger.json

```diff
@@ -151,12 +156,80 @@
                 "$ref" : "#/definitions/Issue"
               }
             }
+          }
+        }
+      }
+    },
+    "/issue/paginate/{start-end}" : {
```

Comment:
- I think a single GET action named '/issues' can be enough.
- It can have two parameters: start and count
- Parameters can have default values if not specified.
- It is better to have these as query parameters: see https://swagger.io/docs/specification/describing-parameters/